### PR TITLE
Add libpqxx

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-c-cpp - this is ready for review. Thanks!

File: recipes/libpqxx/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Project overrides
```
Comment (xhochy) [inline recipes/libpqxx/conda_build_config.yaml:1]: Could using a newer SDK be enough? See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=macosx_sdk_version#requiring-newer-macos-sdks

Comment (Tobias-Fischer) [reply recipes/libpqxx/conda_build_config.yaml:1]: It doesn't seem like it: 
```
FAILED: test/CMakeFiles/runner.dir/test26.cxx.o 
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -DPQXX_SHARED -I$PREFIX/include/server -Iinclude -I../include -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/libpqxx-7.4.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -O3 -DNDEBUG -isysroot /Applications/Xcode_12.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -mmacosx-version-min=10.9 -std=c++17 -MD -MT test/CMakeFiles/runner.dir/test26.cxx.o -MF test/CMakeFiles/runner.dir/test26.cxx.o.d -o test/CMakeFiles/runner.dir/test26.cxx.o -c ../test/test26.cxx
../test/test26.cxx:47:21: error: 'value' is unavailable: introduced in macOS 10.13
      conversions[y.value()] = To4Digits(y.value());
```

Shall I go back to what I had before?

Comment (xhochy) [reply recipes/libpqxx/conda_build_config.yaml:1]: Yes but please include a comment why .

Comment (Tobias-Fischer) [reply recipes/libpqxx/conda_build_config.yaml:1]: Cool, just done this @xhochy.

**Comment by Tobias-Fischer** [comment]
This is ready for re-review @xhochy

**Pull request merged**